src/main/java/mdlaf/MaterialLookAndFeel.java

```diff
@@ -113,7 +113,7 @@ public class MaterialLookAndFeel extends MetalLookAndFeel {
   private static final String separatorUI = MaterialSeparatorUI.class.getCanonicalName();
   private static final String fileChooserUI = MaterialFileChooserUI.class.getCanonicalName();
   private static final String toolTipUI = MaterialToolTipUI.class.getCanonicalName();
-  private static final String taskPaneUI = MaterialTaskPaneUI.class.getCanonicalName();
```

Comment:
I suspect that there is an import to remove too right?

Then we should remove the package inside the gradle build system, otherwise iirc this will works always because you are always importing the `swingx` library with our build system

Reply:
I'm not sure if I understand the question. In my testing to see everything worked fine, I made sure only material ui swing and jiconfont was in my classpath, which worked fine. I also tested if it would crash if I called `MaterialTaskPaneUI.class.getCanonicalName()` directly on my program which it does.

The way I left is that material ui still included swingx as dependency and no project using it needs to be changed if they expect it to be present, but it can be manually excluded by other projects and won't crash since NoClassDefFoundError is catched when trying to loadMaterialTaskPaneUI.

Reply:
idk that you are testing nothing useful, the classpath will contains always the swingx https://github.com/vincenzopalazzo/material-ui-swing/blob/master/build.gradle#L29

Reply:
I may not have worded in a good way, I'll to rephrase what I did.
Leave swingx as implementation dependency to avoid breaking any project that uses material ui and expects it to provide swingx.
Consumers may exclude swingx using
```groovy
implementation ('io.github.vincenzopalazzo:material-ui-swing:1.1.2') {
        exclude(group: 'org.swinglabs') // Excludes both swingx and the scheduler library
    }
```
Projects that excluded swingx can now work with those changes.

Reply:
I see, we can add a couple of line inside the readme to tell people that `swingx` can be removed?

Reply:
Sure